# Use BUSCO with default Python instead of Python3
# October 27th 2017 by B. Hajgato (Free Ubiveristy Brussel - VUB)
--- dammit-0.3.2/dammit/tasks.py.orig	2016-11-23 05:04:29.000000000 +0100
+++ dammit-0.3.2/dammit/tasks.py	2017-10-27 13:47:22.687134646 +0200
@@ -376,7 +376,7 @@
     # BUSCO chokes on file paths as output names
     output_name = os.path.basename(output_name)
 
-    cmd = 'python3 {exc} -in {input_filename} -f -o {output_name} -l {busco_db_dir} '\
+    cmd = 'python {exc} -in {input_filename} -f -o {output_name} -l {busco_db_dir} '\
             '-m {input_type} -c {n_threads}'.format(**locals())
 
     return {'name': name,
